features/taxation/managing_tax_rates/adding_tax_rate.feature

```diff
@@ -0,0 +1,24 @@
+@managing_tax_rates
+Feature: Adding a new tax rate
+    In order to apply different tax rates to various tax categories
+    As an Administrator
+    I want to add a new tax rate to the registry
+
+    Background:
+        Given there is "EU" zone containing all members of European Union
+        And the store has a tax category "Food and Beverage"
+        And the store has a default tax calculator defined
+        And I am logged in as an administrator
+
+    @todo
+    Scenario: Adding a new tax rate
+        Given I want to create a new tax rate
+        When I specify its code as "EUROPEAN_UNION_SALES_TAX"
+        And I name it "European Union Sales Tax"
+        And I define it for a "EU" zone
```

Comment:
for the "EU" zone.
 Cause from now on it is already defined in the system and we know which one it is. :D